package.json

```diff
@@ -74,6 +74,7 @@
     "static-server": "node lighthouse-cli/test/fixtures/static-server.js"
   },
   "devDependencies": {
+    "@build-tracker/cli": "^1.0.0-beta.14",
```

Comment:
theres beta 15 now https://github.com/paularmstrong/build-tracker/compare/v1.0.0-beta.14...v1.0.0-beta.15

Reply:
good catch. i actually asked him to publish that one. :)
it fixed the slow responsiveness of the hoverline on https://lh-build-tracker.herokuapp.com/